File: structural/java/adapter/src/Rectangle.java
public class Rectangle implements Shape{
    private double width;
    private double height;

    public Rectangle(double width, double height){
        this.width  = width;
        this.height = height;
    }
    @Override
    public void draw() {
        for(int i=0;i<this.height; i++){
            for(int j=0; j<this.width; j++) {
                System.out.print("*");
            }
            System.out.println();
        }
    }

    @Override
    public double getWidth() {
        return this.width;
    }

    @Override
    public double getHeight() {
        return this.height;
    }
}

File: structural/python/adapter/Rectangle.py
from Shape import Shape
class Rectangle(Shape):
    __width:  float = 0.0
    __height: float = 0.0
    def __init__(self, width, height):
        self.__width  = width
        self.__height = height

    def draw(self) ->None:
        for i in range(self.__height):
            for j in range(self.__width):
                print("*", end="")
            print()

    def getWidth(self) ->float:
        return self.__width

    def getHeight(self) ->float:
        return self.__height